app/Http/Controllers/FrontController.php

```diff
@@ -469,4 +469,64 @@ public function showAtlRamp() {
 	public function showCltRamp() {
         return view('site.ramp')->with('afld','CLT');
     }
+
+    public function pilotGuideAtl() {
+        $atc = ATC::get();
+        if($atc) {
+            $lcl_controllers = array();
+            $app_controllers = array();
+            $ctr_controllers = array();
+            foreach($atc as $a) {
+                $field = substr($a->position, 0, 3);
+                $position = substr($a->position, -3);
+                if(($field == 'ATL')||($field == 'ZTL')) {
+                    if($position == 'TWR' || $position == 'GND' || $position == 'DEL') {
+                        $lcl_controllers[] = $a;
+                    } elseif($position == 'APP' || $position == 'DEP') {
+                        $app_controllers[] = $a;
+                    } elseif($position == 'CTR') {
+                        $ctr_controllers[] = $a;
+                    }
+                } 
+            }
+        }
+        if(count($lcl_controllers) < 1) {
+            $lcl_controllers = $app_controllers;
+            if(count($lcl_controllers) < 1) {
+                $lcl_controllers = $ctr_controllers;
+            }
+        }
+        $last_update = ControllerLogUpdate::orderBy('id', 'desc')->first();
+        $controllers_update = substr($last_update->created_at, -8, 5);
+
+        $client = new Client(['http_errors' => false]);
+        $icao_id = 'KATL';
+        $res = $client->request('GET', 'https://api.aviationapi.com/v1/charts?apt=' . $icao_id);
+        $status = $res->getStatusCode();
+        $diag = $aaup = '#';
+        if($status == 404) {
+            $charts = null;
+        } elseif(json_decode($res->getBody()) != '[]') {
+            $charts = collect(json_decode($res->getBody())->$icao_id);
+            $diag = $charts->where('chart_code', 'APD');
```

Comment:
OR just change this to `if($status == 200 && ...)` then remove the first part of the if statement. Good practice to only have the same actions in one place in flow control (i.e. you have the same thing in the `if` and the `else` - try to avoid that so if you decide to change that action in the future you can just do it in one place).

Reply:
As said above, fixed per your suggestion by combining the if and elseif conditions into the same logic check.